cmake/config.cmake

```diff
@@ -73,6 +77,11 @@ if(OMR_LLJB)
 	set(OMR_LLJB_TEST ON CACHE BOOL "Enable lljb tests")
 endif()
 
+if (OMR_AOT)
+	add_definitions(-DOMR_RELOCATION_RUNTIME)
+	set(OMR_AOT_TEST ON CACHE BOOL "Enable lljb tests")
```

Comment:
Is that string correct?

Reply:
Thank you for the comment, I've fixed that mistake.